components/src/core/components/Input/Time/TimePicker.vue

```diff
@@ -218,10 +208,15 @@ export default defineComponent({
       setValue(newValue, type);
     };
 
-    const onInput = e => {
-      enteredValue = e.target.value.replace(/\D/g, '');
-      if (enteredValue.length > 2) {
-        enteredValue = enteredValue.substring(0, 2);
```

Comment:
Here enteredValue can be the minute or the hour value.
Need to save them separately